bin/read-config.js

```diff
@@ -1,17 +1,30 @@
 const chalk = require('chalk');
 const fs = require('fs');
 const path = require('path');
+const merge = require('lodash.merge');
```

Comment:
Using this is hacky, often hides intent, easily leads to mistakes (because things go wrong can silently). I veto introducing this.

Reply:
done